ffmuc-ipv6-ra-filter/Makefile

```diff
@@ -0,0 +1,14 @@
+include $(TOPDIR)/rules.mk
```

Comment:
Package License is missing.

Reply:
As discussed in Mattermost, we will be using `MIT`.
cc @awlx @goligo @mayrstefan